extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.css

```diff
@@ -90,23 +91,33 @@ iframe.swg-dialog {
   font-weight: 400 !important;
 }
 
-/* TODO(#38112): confirm with UX about if there's better animation to be used. */
+[dir="rtl"] .i-amphtml-story-subscriptions-dialog-banner-button {
+  transform: translateX(0.2em) !important;
+}
+
 .i-amphtml-story-subscriptions-dialog-banner-button.i-amphtml-story-subscriptions-dialog-banner-button-visible {
   opacity: 1 !important;
+  transform: translateX(0) !important;
 }
 
 .i-amphtml-story-subscriptions-dialog-banner-button:after {
   content: '' !important;
   width: 0.93em !important;
   height: 0.93em !important;
   margin: 0 0.6em !important;
+  transform: translateX(-0.1em) !important;
+  transition: transform 0.3s ease-out !important;
   background-size: contain !important;
   background-repeat: no-repeat !important;
   background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" stroke="black" fill="none" stroke-width="2px" stroke-linecap="round" stroke-linejoin="round"><polyline points="1.75,1.5 6,6 1.75,10.6"/></svg>') !important;
 }
 
 [dir="rtl"] .i-amphtml-story-subscriptions-dialog-banner-button:after {
-  transform: scale(-1, 1) !important;
+  transform: scale(-1, 1) translateX(0.1em) !important;
```

Comment:
The translate gets flipped since the scale is flipped. The translate so it needs to be a negative value.
Or change the order, in transform order matters so this will work like `translateX(0.1em) scaleX(-1)`, otherwise 
the scale will be factored into the translate.

Reply:
Nice catch! I didn't tell the differences at all. And this is good, this use case demonstrated a lot of transition/transform insights!